client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/DataFrameTransformConfig.java

```diff
@@ -40,6 +44,8 @@
     public static final ParseField SOURCE = new ParseField("source");
     public static final ParseField DEST = new ParseField("dest");
     public static final ParseField DESCRIPTION = new ParseField("description");
+    public static final ParseField TRANSFORM_VERSION = new ParseField("transform_version");
```

Comment:
I wonder if we should call this just `version`. We went for `job_version` for anomaly detection jobs in hindsight it seems redundant.

Reply:
Eh, I am fine with changing it.